test/test.butt

```diff
@@ -0,0 +1,118 @@
+#!/bin/env butt
+
+#######################################
+## Basic
+#######################################
+# start "test comment"
+#   assert_equal 1 0
+# end
+#######################################
+start "test no arguments"
```

Comment:
Please, improve the test case message, e.g. "running butt without arguments fails". Applies to other tests as well.